### PR TITLE
test(client): Tracing Links

### PR DESCRIPTION
**Pull request by danstarns**
Adds test back after engine fix.

### CODE REVIEW

**Comment by danstarns** [comment]
related: https://github.com/prisma/prisma-engines/pull/3101

File: packages/client/tests/functional/tracing/tests.ts [hunk 1]
```diff
@@ -114,6 +114,7 @@ testMatrix.setupTestSuite(({ provider }) => {
       if (key === 'resource') return undefined
       if (key === 'spanId') return '<spanID>'
```
Comment (millsp) [inline packages/client/tests/functional/tracing/tests.ts:115]:
```suggestion
      if (key === 'spanId') return '<spanId>'
```

File: packages/client/tests/functional/tracing/tests.ts [hunk 2]
```diff
@@ -114,6 +114,7 @@ testMatrix.setupTestSuite(({ provider }) => {
...
       if (key === 'traceId') return '<traceId>'
+      if (key === 'links') return value.map(() => ({ traceId: '<traceId>', spanId: '<spanId>' }))
```
Comment (millsp) [inline packages/client/tests/functional/tracing/tests.ts:117]:
```suggestion
```

Comment (millsp) [reply packages/client/tests/functional/tracing/tests.ts:117]: It's already covered by the two lines above, I believe.

**Review by millsp — CHANGES_REQUESTED** [review]

**Review by millsp — APPROVED** [review]

**Pull request merged**